src/swagger.yml

```diff
@@ -2963,6 +3243,232 @@ definitions:
          articles:
            $ref: '#/definitions/ScreenArticles'
 
+  ### Appraisal
+  Appraisal:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/UUIDv4'
+      created_on:
+        type: string
+        format: date-time
+        description: Timestamp
+      edited_on:
+        type: string
+        format: date-time
+        description: Timestamp
+      edited_by:
+        type: string
+        description: Editor's name
+      wine_year:
+        $ref: '#/definitions/Year'
+      title:
+        type: string
+        description: Title
+      description:
+        type: string
+        description: Description
+      original_title:
+        type: string
+        description: Title before edit
+      original_description:
+        type: string
+        description: Description before edit
+      rating:
+        type: integer
+        format: int32
+        minimum: 1
+        maximum: 5
+        description: Given rating
+      display_name:
+        type: string
+        description: Person's (public) name
+      # @todo Could use all-of SweepstakeParticipant for person's fields
+      firstname:
+        type: string
+        description: Person's firstname
+      lastname:
+        type: string
+        description: Person's lastname
+      address:
+        type: string
+        description: Person's address
+      zip_code:
+        type: integer
+        format: int32
+        minimum: 1000
+        maximum: 9699
+        description: Person city's ZIP code
+      city:
+        type: string
+        description: Person's city
+      email:
+        type: string
+#        pattern: @todo Fixed pattern?
+        description: Person's (public) e-mail address
+      gender:
+        type: string
+#        pattern: '/^(male|female)$/'
```

Comment:
It would work, questio s if we should .. ich that case an "enum" with the 2 possible values would be better:
```
enum:
 - "female"
 - "male"
```